include/deal.II/grid/grid_tools.h

```diff
@@ -157,10 +158,27 @@ namespace GridTools
    * get the same result using <code>cell-@>measure()</code>, but this
    * function also works for cells that do not exist except that you make it
    * up by naming its vertices from the list.
+   *
+   * Instantiations of this function are available for the <tt>dim !=
+   * spacedim</tt> cases, but these just raise exceptions. They are only
+   * provided as an aid for generic programming.
+   */
+  template <int dim, int spacedim>
```

Comment:
on another hand, if we switch to C++11 next release, the remaining 3.8% may actually finally start using it as well :smile: